packages/thumbprint-atomic/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Added
+
+-   [Minor] Add `-100` versions of colours as background color classes. ([#809](https://github.com/thumbtack/thumbprint/issues/809))
```

Comment:
```suggestion
-   [Minor] Add `-100` versions of colours as background color classes. (#809)
```

Let's do just the issue number for consistency with other entries. Although I agree that having the link is useful. Maybe that can be a different PR.

Reply:
Fair enough. This is actually a feature of GitLens, it autosuggested this link when i typed the issue number 🤯